Pollo/Extensions/NotificationBanner+Shared.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  NotificationBanner+Shared.swift
+//  Pollo
+//
+//  Created by Jack Thompson on 2/4/20.
+//  Copyright © 2020 CornellAppDev. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import NotificationBannerSwift
+
+extension NotificationBanner {
+
+    static func connectingBanner() -> BaseNotificationBanner {
+        let banner = StatusBarNotificationBanner(title: "Connecting...", style: .warning, colors: PolloBannerColors())
+        banner.autoDismiss = false
+        return banner
+    }
+
+    /// Displayed when a connection is successfully established
```

Comment:
Why do some comments have /// and others only have // ?

Reply:
`///` is used for comments outside functions

Reply:
^^ Yeah i wanted to add some documentation spec for those functions